lib/SSO.php

```diff
@@ -28,6 +29,7 @@ public function getAuthorizationUrl(
         $state,
         $provider,
         $connection,
+        $organization,
```

Comment:
I think in order to make this a non-breaking change we'll need to default this to null.

```suggestion
        $organization = null,
```
```suggestion
        $organization,
```